driver/src/orderbook/mod.rs

```diff
@@ -8,6 +8,7 @@ use mockall::automock;
 use paginated_auction_data_reader::PaginatedAuctionDataReader;
 use std::convert::TryInto;
 
+mod event_based_orderbook;
```

Comment:
uber-naming-nit: `orderbook::event_based_orderbook` seams like a slightly redundant name to me (same for `filtered_orderbook`). How about `orderbook::streamed`?